bin/all_sky_search/pycbc_strip_injections

```diff
@@ -27,7 +27,7 @@ pycbc.init_logging(args.verbose)
 
 logging.info('File: %s' % args.injection_file)
 indoc = ligolw_utils.load_filename(args.injection_file, False, contenthandler=LIGOLWContentHandler)
-sim_table = table.get_table(indoc, 'sim_inspiral')
+sim_table = table.Table.get_table(indoc, 'sim_inspiral')
```

Comment:
Apart from minimizing the amount of code change, is there any other reason why this could not be
```
sim_table = lsctables.SimInspiralTable.get_table(indoc)
```
as was done for the other cases?

Reply:
There's a mix of both use cases throughout PyCBC. I didn't change this in any instance (if the names change, we have a bigger problem).